### ISSUE TITLE
"Yes" and "No" do not remain as strings

### ISSUE THREAD

**Issue by BananaNeil** [HipByte/motion-yaml#14]
My app needs to be able to store the string `"Yes"` and the string `"No"` in yaml.. but when I pull them back out, they are replaced with `true` and `false`

```
>> YAML.load('yes'.to_yaml)
=> true

>> YAML.load(["No"].to_yaml)
=> [false]

>> YAML.load(["yeS"].to_yaml)
=> [true]
```

The same happens with `"true"` and `"false"`.

I don't think this is the expected behavior?
It is not how the pure ruby yaml gem works.

**Comment by Watson1978** [comment]
Which version did you use ?
If you use v1.6 (latest version), you can solve your issue.

Thank you.

**Comment by BananaNeil** [comment]
I am using v1.6, the problem still exists.

**Comment by Watson1978** [comment]
OK I see. There is a difference in where dump YAML.

```
(main)> YAML.dump(["No"])
=> "- No\n"
(main)> ["No"].to_yaml
=> "- No\n"
```

**Comment by Watson1978** [comment]
This issue should be fixed with v1.6.1.
Thank you
